### PR TITLE
Added Chat and Comuna pages

### PR DESCRIPTION
**Pull request by YuraGora**

### CODE REVIEW

File: pages/Chat.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+from selenium.webdriver.common.by import By
+
+
+class ChatPage:
+    """
+        Locators and methods for Chat Page.
+    """
+    CHAT_SMALL_AVATAR_CLASS_NAME = "SmallAvatar"
+    MESSAGE_TEXTAREA_XPATH = "//div/textarea"
+    SEND_BUTTON_XPATH = "//button[@type='submit']"
```
Comment (vladskakun) [inline pages/Chat.py:11]: You can change 'button' to 'btn'. It will be more readable.

File: pages/Chat.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
```
Comment (mehalyna) [inline pages/Chat.py:12]: don't forget about docstrings to each method

File: pages/Comuna.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+
+class ComunaPage:
+    """
+        Locators and methods for Comuna page.
+    """
+    NAV_MENU_COMUNA_XPATH = "//span[text() = 'Comuna']"
```
Comment (mehalyna) [inline pages/Comuna.py:9]: rename NAV_MENU_COMUNA_XPATH according to the type of UI element.
COMUNA_NAV_MENU_ITEM_XPATH
or
COMUNA_NAV_MENU_LINK_COMUNA_XPATH

File: pages/Comuna.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    COMUNA_USER_CSS = "div:nth-child({}) > a > div > div > h5"
```
Comment (mehalyna) [inline pages/Comuna.py:10]: rename COMUNA_USER_CSS according to the type of UI element.
COMUNA_USER_ITEM_CSS
or
COMUNA_USER_LINK_CSS
or 
COMUNA_USER_BTN_CSS

File: pages/Comuna.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+    def __init__(self):
+        self.driver = webdriver.Chrome()
+
+    def click_chat_comuna(self):
+        element = self.driver.find_element(By.XPATH, self.NAV_MENU_COMUNA_XPATH)
+        element.click()
+
+    def click_chat_comuna_user(self, number):
```
Comment (TanyaKalistruk) [inline pages/Comuna.py:19]: `click_comuna_user_chat` sounds better

File: pages/Comuna.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+        element = self.driver.find_element(By.CSS_SELECTOR, self.COMUNA_USER_CSS.format(number), number)
```
Comment (TanyaKalistruk) [inline pages/Comuna.py:20]: Good job with CSS locator :+1:

File: pages/Chat.py (hunk 1 above)
Comment (YuraGora) [reply pages/Chat.py]: Done

File: pages/Chat.py (hunk 2 above)
Comment (YuraGora) [reply pages/Chat.py:12]: Done

File: pages/Comuna.py (hunk 3 above)
Comment (YuraGora) [reply pages/Comuna.py]: Done

File: pages/Comuna.py (hunk 4 above)
Comment (YuraGora) [reply pages/Comuna.py]: Done

File: pages/Chat.py (hunk 2 above)
Comment (YuraGora) [reply pages/Chat.py:12]: Thank You! :)

File: pages/Comuna.py (hunk 5 above)
Comment (YuraGora) [reply pages/Comuna.py]: Ok, Done!

File: pages/Comuna.py (hunk 6 above)
Comment (YuraGora) [reply pages/Comuna.py]: Thank You! :)

File: pages/Chat.py [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+        element = self.driver.find_element(By.CSS_SELECTOR, self.MESSAGE_TEXTAREA_CSS)
+        element.click()
+
+    def send_textarea(self, message):
+        """ Method for type message in text area 'Type your message...' """
+        element = self.driver.find_element(By.CSS_SELECTOR, self.MESSAGE_TEXTAREA_CSS)
+        element.send_keys(message)
+
+    def click_send_button(self):
+        """ Method for click "Send" button """
```
Comment (mehalyna) [inline pages/Chat.py:32]: use multiline docstrings

File: pages/Chat.py [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+        element = self.driver.find_element(By.CLASS_NAME, self.CHAT_SMALL_AVATAR_BTN_CLASS_NAME)
+        element.click()
+
+    def click_textarea(self):
+        """ Method for click on textarea 'Type your message...' """
+        element = self.driver.find_element(By.CSS_SELECTOR, self.MESSAGE_TEXTAREA_CSS)
+        element.click()
+
+    def send_textarea(self, message):
+        """ Method for type message in text area 'Type your message...' """
```
Comment (mehalyna) [inline pages/Chat.py:27]: use multiline docstrings

File: pages/Chat.py [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+    def __init__(self):
+        self.driver = webdriver.Chrome()
+
+    def click_chat_small_avatar(self):
+        """ Method for click small avatar on chat page."""
+        element = self.driver.find_element(By.CLASS_NAME, self.CHAT_SMALL_AVATAR_BTN_CLASS_NAME)
+        element.click()
+
+    def click_textarea(self):
+        """ Method for click on textarea 'Type your message...' """
```
Comment (mehalyna) [inline pages/Chat.py:22]: use multiline docstrings

File: pages/Comuna.py [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+    def __init__(self):
+        self.driver = webdriver.Chrome()
+
+    def click_chat_comuna(self):
+        element = self.driver.find_element(By.CSS_SELECTOR, self.COMUNA_NAV_MENU_ITEM_CSS)
+        element.click()
+
+    def click_comuna_user_chat(self, number):
+        element = self.driver.find_element(By.CSS_SELECTOR, self.COMUNA_USER_ITEM_CSS.format(number))
+        element.click()
```
Comment (mehalyna) [inline pages/Comuna.py:21]: methods should each have their own docstrings

File: pages/Comuna.py [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+    """
+    COMUNA_NAV_MENU_ITEM_CSS = "li:nth-child(2) > a > span > span.nav-item-text"
+    COMUNA_USER_ITEM_CSS = "div:nth-child({}) > a > div > div > h5"
+
+    def __init__(self):
+        self.driver = webdriver.Chrome()
+
+    def click_chat_comuna(self):
+        """
+            Method for click in navigation menu on Comuna.
```
Comment (TanyaKalistruk) [inline pages/Comuna.py:17]: Method for click in navigation menu on Comuna item.

**Review by TanyaKalistruk — APPROVED** [review]

File: pages/Chat.py (hunk 7 above)
Comment (YuraGora) [reply pages/Chat.py]: Ok, done.

File: pages/Chat.py (hunk 8 above)
Comment (YuraGora) [reply pages/Chat.py]: Done.

File: pages/Chat.py (hunk 9 above)
Comment (YuraGora) [reply pages/Chat.py]: Done.

File: pages/Comuna.py (hunk 10 above)
Comment (YuraGora) [reply pages/Comuna.py]: Done, thank you!

File: pages/Comuna.py (hunk 11 above)
Comment (YuraGora) [reply pages/Comuna.py]: Ok, thank you!

**Review by mehalyna — APPROVED** [review]

**Review by vladskakun — APPROVED** [review]

**Pull request merged**